codes/classical/properties/codes_for_distributed_storage.yml

```diff
@@ -0,0 +1,21 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: codes_for_distributed_storage
```

Comment:
distributed_storage

Reply:
I've created an entry for this already; no need to resubmit unless you'd like to expand.